hearpreprocess/pipeline.py

```diff
@@ -831,7 +913,9 @@ def run(self):
                 ).open("w"),
                 indent=True,
             )
-            labelset = labelset | set(labeldf["label"].unique().tolist())
+            split_labelset = set(labeldf["label"].unique().tolist())
+            assert len(split_labelset) != 0
+            labelset = labelset | split_labelset
```

Comment:
Was the error that we stopped creating the csv files with the labels above? -- maybe we should add an assert that makes sure that there are the correct number of csv files available in the `list(self.requires()["subcorpus_metadata"].workdir.glob("*.csv"))` expression in at the start of the for loop so we don't silently fail on this?

Reply:
change from reading with split name so this issue doesnot arise